src/components/header/index.jsx

```diff
@@ -0,0 +1,162 @@
+import {
+  Box,
+  Stack,
+  Typography,
+  IconButton,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemText,
+  ListItemIcon,
+  Divider,
+} from "@mui/material";
+import { FiSearch } from "react-icons/fi";
+import DefaultInput from "../inputs/default-input";
+import Logo from "../../assets/icons/logo.svg";
+import heartIcon from "../../assets/icons/heart-primary.svg";
+import userIcon from "../../assets/icons/user.svg";
+import bagIcon from "../../assets/icons/bag.svg";
+import NavLink from "../links/nav-link";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import ButtonWithIcon from "../buttons/button-with-Icon";
+import MenuIcon from "@mui/icons-material/Menu";
+import Drawer from "@mui/material/Drawer";
+import { useState } from "react";
+
+const Header = ({ Categories }) => {
```

Comment:
do you want to use this prop (categories), inside your code?
As I see it's not used

Reply:
Done